assets/css/_faq.scss

```diff
@@ -0,0 +1,53 @@
+.m-faq {
+  background: $color-bg-medium;
+
+  @media (min-width: 720px) {
+    height: 100%;
+    position: fixed;
+    width: 100%;
+  }
+}
+
+.m-faq__container {
+  background: $color-bg-light;
+  padding: 2rem;
+
+  @media (min-width: 720px) {
+    width: 50%;
+    height: 100%;
+    position: fixed;
+  }
+
+  @media (min-width: 1200px) {
+    width: 33%;
```

Comment:
A single `max-width` would be simpler than the media queries and would probably achieve the same goal.

Reply:
the problem is on phone-sized screens -- those screens are narrow enough that it looks weird if this page's content isn't full width.

Reply:
What about `width: 100%; max-width: 600px;`. That'd fill phone screens, wouldn't it? And then the margin on the right would only appear if the screen gets big enough that it's not weird?

Reply:
oh, you're right. i was thinking of using percentages but we can just use a set width.